modules/serverless-custom-revision-urls.adoc

```diff
@@ -0,0 +1,33 @@
+[id="serverless-custom-revision-urls"_{context}"]
+= Creating and removing custom URLs for revisions by using tags
+
+Assigning a `--tag` flag to a service by using the `kn service update` command creates a custom URL for the revision that is created when the service is updated. The custom URL is given in the format `http(s)://<tagged_service_name>.<domain>`.
```

Comment:
Here `<tagged_service_name>` I see it actually creates in the format  `<tag>-<service_name>-<namespace>`. Example, give a tag v2 and service helloworld on default namespace would resolve to `http(s)://v2-helloworld-default.<domain>`

Reply:
Updated, thank you!